<?xml version='1.0'?>
<launch>
	<!-- Processor Node -->
	<node pkg="egh450_image_processor" type="processor" name="processor" clear_params="true" output="screen">
		<param name="cascade_file" value="$(find egh450_image_processor)/data/btd20_06102019_Haar2.xml"/>
		<param name="triangle_radius" value="0.156" type="double"/>
		<param name="square_radius" value="0.152" type="double"/>
		<!-- Setting this to true will tell the program to use "~image_raw" + "/compressed" instead -->
		<param name="use_compressed" value="true" type="bool"/>

		<!-- Need to add HSV params -->

		<!-- Topic remappings to connect to the correct camera -->
		<remap from="~camera_info" to="/raspicam_node/camera_info"/>
		<remap from="~image_raw" to="/raspicam_node/image_raw"/>
		<remap from="~image_raw/compressed" to="/raspicam_node/image/compressed"/>
	</node>
</launch>
